src/packages/@ncigdc/modern_components/GeneExpression/GeneExpression.js

```diff
@@ -22,6 +22,30 @@ import GeneExpressionChart from './GeneExpressionChart';
 // import mockData from './inchlib/data';
 import * as helper from './helpers';
 
+const featureNamesAllowList = [
+  // alphabetized
+  'age_at_diagnosis',
+  'case_id',
+  'ethnicity',
+  'gender',
+  'race',
+  'submitter_id',
+  'vital_status',
+];
```

Comment:
Future enhancement: make this a deny list instead, so back end can add whatever additional features, without our needing to add them here.

Reply:
The alphabetisation can then be done programatically using `sort` on the incoming object.